car_manager.py

```diff
@@ -0,0 +1,38 @@
+import random
+from turtle import Turtle
+COLORS = ["red", "orange", "yellow", "green", "blue", "purple"]
+STARTING_MOVE_DISTANCE = 5
+MOVE_INCREMENT = 10
+
+
+class CarManager(Turtle):
+
+    def __init__(self):
+        super().__init__()
+        self.all_cars = []
+        self.create_car()
+        self.speed = 10
```

Comment:
consider making the speed user definable rather than making it fixed. That way the speed can be adjusted easily if the need arises.